low-level/framework/base/sspl_constants.py

```diff
@@ -207,7 +207,7 @@
 
 class RaidDataConfig(Enum):
     MDSTAT_FILE = "/proc/mdstat"
-    DIR = "/sys/block/"
+    BLOCK_DIR = "/block/"
```

Comment:
This is to build /sys/block eventually i guess. It doesnt need to limit to RAID Config, keep generic and make all dependent sensor to use from this new constant

Reply:
yes that is to eventually build /sys/block/, but don't see use of /block dir anywhere else in sspl.

Reply:
not today, but in future it would be needed, so bring it out of RAID context